.gitignore

```diff
@@ -59,3 +59,10 @@ outdated/
 perf-test.py
 *.prof
 *.so
+
+# example
+/examples/example.yaml
```

Comment:
It's better to have the example yaml be tracked inside the repo. You can have your own testing folder in other places on your local machine.